src/components/SearchResults.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/SearchResults.css";
+
+const SearchResults = ({ results }) => {
+  return (
+    <div className="SearchResults" data-testid="SearchResults">
+        <img className="card-image" data-testid="card-image" src={results} alt="spaceImage" />
+    </div>
+  );
+}
+
+
+SearchResults.propTypes = {
+  image: PropTypes.object,
```

Comment:
Why do we have an image prop that we are not using? If you don't need it, get rid :)